src/foam/dao/ManyToManyRelationshipDAO.js

```diff
@@ -18,51 +18,38 @@
 foam.CLASS({
   package: 'foam.dao',
   name: 'ManyToManyRelationshipDAO',
-  extends: 'foam.dao.RelationshipDAO',
+  extends: 'foam.dao.ProxyDAO',
 
   implements: [ 'foam.mlang.Expressions' ],
 
   documentation: 'Adapts a DAO based on a *:* Relationship.',
 
   properties: [
```

Comment:
... we should also include documentation about what this DAO is `of`, and what it's `delegate` is `of`. They used to be `of` different things (which was a bit confusing, but workable).